app/js/sumo_db.js

```diff
@@ -154,6 +166,29 @@
       endpoint += '?format=json'; // TODO bug 1088014
 
       return request(endpoint, 'GET').then(JSON.parse);
+    },
+
+    update_settings: function(user, setting) {
```

Comment:
I was waiting on feedback on the below so that I could address all three in one commit.